apps/auth/main.py

```
@@ -0,0 +1,40 @@
+    description="Authentication microservice for Envybase",
+    version="0",
+)
+
+
+@app.get("/")
+def read_root():
+    return {"status": "healthy", "service": "auth"}
+
+
+@app.get("/frontendinfo", summary="Get frontend info")
+def frontendinfo():
```

Ruff (0.8.2)

Missing return type annotation for public function `frontendinfo`

(ANN201)